src/main/resources/application.yaml

```diff
@@ -32,9 +32,9 @@ spring:
     name: Civil SDT
   datasource:
     driver-class-name: org.postgresql.Driver
-    url: jdbc:postgresql://${SDT_DB_HOST:localhost}:${SDT_DB_PORT:5432}/${SDT_DB_NAME:civil_sdt}${SDT_DB_OPTIONS:}
+    url: jdbc:postgresql://${SDT_DB_HOST:localhost}:${SDT_DB_PORT:5055}/${SDT_DB_NAME:civil_sdt}${SDT_DB_OPTIONS:}
```

Comment:
The default SDT_DB_PORT value is incorrect - should be 5432 rather than 5055.

Reply:
done